bftengine/src/bftengine/BFTEngine.cpp

```diff
@@ -182,6 +182,7 @@ IReplica::IReplicaPtr IReplica::createNewReplica(const ReplicaConfig &replicaCon
 
     // Init the persistent storage
     ((PersistentStorageImp *)persistentStoragePtr.get())->init(move(metadataStoragePtr));
+    persistentStoragePtr->setNewEpochFlag(false);
```

Comment:
set to false the nre epoch flag (to handler cases where we don't remove the metadata)